pkg/validator/app.go

```diff
@@ -55,6 +55,16 @@ func (obj appPatchValue) MarshalJSON() ([]byte, error) {
 	return json.Marshal(obj.stringSliceValue)
 }
 
+func validateAppName(
```

Comment:
IMO we should just add maxLength to the metadata.name in the kdapp CR, rather than dynamically checking in the webhook.

Reply:
I.e. enhance the current "metadata" part of the kdapp validation to be:
```yaml
        metadata:
          type: object
          properties:
            name:
              type: string
              maxLength: 63
```